platform/linux-generic/include-abi/odp/api/abi/packet.h

```diff
@@ -17,6 +17,7 @@ extern "C" {
 #endif
 
 #include <odp/api/plat/packet_inlines.h>
+#include <odp/api/plat/event_vector_inlines.h>
```

Comment:
This belongs to `platform/linux-generic/include-abi/odp/api/abi/event_vector.h`

Reply:
moved in v12